_api/hasura/actions/_handlers/updateBigQuestion.ts

```diff
@@ -0,0 +1,79 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { z } from 'zod';
+
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import { getInput } from '../../../../api-lib/handlerHelpers';
+
+const updateBigQuestionInput = z
+  .object({
+    big_question_id: z.number().optional().nullable(),
+    cover_image_url: z.string(),
+    css_background_position: z.string().optional().nullable(),
+    description: z.string().optional(),
+    expire_at: z.string().optional(),
+    prompt: z.string(),
+    publish_at: z.string().optional(),
+  })
+  .strict();
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  const {
+    payload: {
+      big_question_id,
+      cover_image_url,
+      css_background_position,
+      description,
+      expire_at,
+      prompt,
+      publish_at,
+    },
+    session: { hasuraProfileId: profileId },
+  } = await getInput(req, updateBigQuestionInput);
+
+  if (profileId) {
+    //TODO replace with team members Ids check
+    console.log(true);
+  }
+  if (big_question_id) {
+    await adminClient.mutate(
+      {
+        update_big_questions_by_pk: [
+          {
+            pk_columns: { id: big_question_id },
+            _set: {
+              cover_image_url: cover_image_url,
```

Comment:
would be ideal if we had a way to upload an image inthe UI, and it would uplaod to cloudflare

like this: generateOneTimeUpload.ts @sshmm